client/object_delete.go

```diff
@@ -118,32 +90,36 @@ func (x ResObjectDelete) Tombstone() oid.ID {
 //
 // Return errors:
 //   - global (see Client docs)
-//   - [ErrMissingContainer]
-//   - [ErrMissingObject]
 //   - [ErrMissingSigner]
 //   - [apistatus.ErrContainerNotFound]
 //   - [apistatus.ErrObjectAccessDenied]
 //   - [apistatus.ErrObjectLocked]
 //   - [apistatus.ErrSessionTokenExpired]
-func (c *Client) ObjectDelete(ctx context.Context, prm PrmObjectDelete) (*ResObjectDelete, error) {
-	switch {
-	case prm.addr.GetContainerID() == nil:
-		return nil, ErrMissingContainer
-	case prm.addr.GetObjectID() == nil:
-		return nil, ErrMissingObject
-	}
+func (c *Client) ObjectDelete(ctx context.Context, containerID cid.ID, objectID oid.ID, prm PrmObjectDelete) (*ResObjectDelete, error) {
```

Comment:
Do we want cid/oid or `oid.Address` here? I tend to think CID/OID pair can be easier to handle in most of the code, but maybe we have any reason to with `Address`?